admin/metabox/class-metabox.php

```diff
@@ -762,6 +769,10 @@ public function save_postdata( $post_id ) {
 			$data       = null;
 			$field_name = WPSEO_Meta::$form_prefix . $key;
 
+			if ( ! isset( $meta_box['type'] ) ) {
+				ray( $meta_box )->red();
```

Comment:
This looks like: https://github.com/spatie/global-ray

Looks like leftover debug code.